rfcs/experimental/README.md

```diff
@@ -5,19 +5,20 @@ released as experimental features in the library. An experimental
 feature is expected to have an implementation that is of comparable quality
 to a fully supported feature. Sufficient tests are required.
 
-An experimental feature does not yet appear as part of the oneDPL 
+An experimental feature does not yet appear as part of the oneDPL
 specification. Therefore, the interface and design can change.
 There is no commitment to backward compatibility for experimental features.
 
 The documents in this directory should include a list of the exit conditions
 that need to be met to move the functionality from experimental to fully supported.
-These conditions might include demonstrated performance improvements, 
-demonstrated interest from the community,
-acceptance of the required specification changes, etc. 
+These conditions might include demonstrated performance improvements, demonstrated
+interest from the community, acceptance of the required specification changes, etc.
 
-For features that require specification changes, the document might
-include wording for those changes or a link to any PRs opened
-against the specification.
+A document here needs to be updated if the corresponding feature undergoes
+modifications while remaining experimental.
```

Comment:
Should you add that the document may also be updated as the status of the listed exit conditions change. For example, if performance improvements have been demonstrated.

Reply:
Thanks; I added a sentence for this.